enterprise/backend/test/metabase_enterprise/serialization/v2/e2e_test.clj

```diff
@@ -735,3 +735,48 @@
                     (is (thrown-with-msg? Exception #"Please upgrade"
                                           (cmd/v2-load! dump-dir {}))
                         "throws")))))))))))
+
+(deftest pivot-export-test
+  (testing "Pivot table export and load correctly"
+    (ts/with-random-dump-dir [dump-dir "serdesv2-"]
+      (ts/with-source-and-dest-dbs
+        (ts/with-source-db
+          (t2.with-temp/with-temp
+            [Database   {db-id :id}             {:name "Pivot Database"}
+             Table      {table-id :id}          {:name "Pivot Table"}
+             Field      {pf1 :id}               {:name "Pivot Field 1"}
+             Field      {pf2 :id}               {:name "Pivot Field 2"}
+             Field      {field-id :id}          {:name "Some Field"}
+             Collection {coll-id :id}           {:name "Pivot Collection"}
+             Card       card                    {:name          "Pivot Card"
+                                                 :collection_id coll-id
+                                                 :dataset_query {:type     :query
+                                                                 :database db-id
+                                                                 :query    {:source-table table-id
+                                                                            :aggregation  [:sum [:field pf1 nil]]
+                                                                            :breakout     [[:field field-id nil]]}}
+                                                 :visualization_settings
+                                                 {:pivot_table.column_split
+                                                  {:columns [["field" pf1 nil]]
+                                                   :rows    [["field" pf2 nil]]
```

Comment:
I wonder how you got this data structure shape. When I tried to reproduce the issue (https://github.com/metabase/metabase/issues/30133) I got this:

```json
    "visualization_settings": {
        "pivot_table.column_split": {
            "rows": [
                [
                    "field",
                    45,
                    {
                        "base-type": "type/Text",
                        "source-field": 43
                    }
                ]
            ],
            "columns": [
                [
                    "field",
                    58,
                    {
                        "base-type": "type/Text",
                        "source-field": 40
                    }
                ]
            ],
            "values": [
                [
                    "aggregation",
                    0
                ],
                [
                    "aggregation",
                    1
                ],
                [
                    "aggregation",
                    2
                ],
                [
                    "aggregation",
                    3
                ],
                [
                    "aggregation",
                    4
                ]
            ]
        }
    },
```